chrome/content/zotero/xpcom/collectionTreeRow.js

```diff
@@ -65,12 +65,15 @@ Zotero.CollectionTreeRow.prototype.__defineGetter__('id', function () {
 		case 'trash':
 			return 'T' + this.ref.libraryID;
 		
+		case 'feeds':
+			// The "0" is kind of weird, but we need the ID to be a one-character prefix + a number
+			// and there's only one set of feeds
```

Comment:
Let's go with "F1" instead of "F0". We already use "L1" for My Library, which is technically the internal libraryID, but still, a little less weird-looking.